Makefile

```
@@ -1,17 +1,17 @@
 # enable cgo to build sqlite
 export CGO_ENABLED = 1
 
-# strip binaries
-FLAGS := -tags sqlite_omit_load_extension,sqlite_json -ldflags '-s'
-
 # default output file
 OUTPUT ?= dbmate
 
 # platform-specific settings
 GOOS := $(shell go env GOOS)
 ifeq ($(GOOS),linux)
 	# statically link binaries to support alpine linux
-	FLAGS := -tags netgo,osusergo,sqlite_omit_load_extension,sqlite_json  -ldflags '-s -extldflags "-static"'
+	override FLAGS := -tags netgo,osusergo,sqlite_omit_load_extension,sqlite_json  -ldflags '-s -extldflags "-static"' $(FLAGS)
```

looks like there was an extra space here in the original file for some reason, may as well fix it now

```suggestion
	override FLAGS := -tags netgo,osusergo,sqlite_omit_load_extension,sqlite_json -ldflags '-s -extldflags "-static"' $(FLAGS)
```